### ISSUE TITLE
CTA / Repository / Create

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#1978]
As a user, I want to be able to **Create** a **Repository** so that I can create or curate content with an Organization or User (my namespace) selected as the owner of it.

## Acceptance criteria

**Entry points**

1. User clicks to create a new repository from an Organization page
2. User completes Create Repository form

Or

1. User Creates a new Repository
2. Selects Organization from the owner list

**Select Repository type**

1. User clicks to Create and new Repository
2. User selects type of Repository they want to create
3. User fills the corresponding form
4. User clicks Create and is taken to the new Repository

Screens for this flow can be found [here](https://zpl.io/GnEgo5m).

## Design docs

- [Organisation page with menu open](https://zpl.io/gg8yrY1)
- [Create Repository screens](https://zpl.io/GnEgo5m)

**Comment by paulsonder** [comment]
@jamlung-ri I think this is feeling pretty good in terms of user flow and page structure.

There are some outstanding content requirements _(for example, on the page where the user chooses which type of Repository to create)_ and probably a need for a detailed definition of the fields for each form and how we organize them (sections). An example of a Collection can be found [here](https://zpl.io/jpZ8J1x).

That said, I think there's enough design to move this forward, if you're in agreement.

**Comment by jamlung-ri** [comment]
Agreed, let's move it forward but leave placeholders for content in the designs, if that makes sense for you. I will work to populate the content once I see how it's presented in the designs.

**Comment by paulsonder** [comment]
@snyaggarwal Take a look through the designs and let me know if you would like to discuss.

**Comment by paulsonder** [comment]
1. That design was intended to show a selected state but I would expect the default would be Source. Though happy to defer to @jamlung-ri on this!
2. Could we create a user story around this and I can add required elements?
3. Added the About card [back](https://zpl.io/jpZ8J1x), must have lost it by mistake.
